infra/base-images/base-runner/coverage

```diff
@@ -448,6 +448,7 @@ elif [[ $FUZZING_LANGUAGE == "jvm" ]]; then
   # overwritten. We need to do this to include all possible classes in
   # the coverage report and avoid duplicate classes from different jar
   # files which jacoco fails to handle.
+  cp $OUT/*.class $classes_dir
```

Comment:
Am a bit unsure why we need this?

Reply:
To ensure the class file for the default package does exist in the classes_dir